src/app/hooks/AuthProvider.tsx

```diff
@@ -68,18 +68,17 @@ export function AuthProvider({ children }: { children: ReactNode }) {
 	const config = useAppConfigContext();
 	const storedToken = getStoredToken();
 	const [egoJwt, setEgoJwt] = useState('');
+	const TokenUtils = createEgoUtils(config.EGO_PUBLIC_KEY);
+	const router = useRouter();
+	const loginStateOnPageLoad = !egoJwt?.length;
```

Comment:
very confusing variable name... loginState i would expect a state as a string from list of options... this is just a bool... could this just be `isLoggedIn = !!egoJwt`?